lib/filewatch/tail.rb

```diff
@@ -42,7 +42,8 @@ def initialize(opts={})
         :stat_interval => 1,
         :discover_interval => 5,
         :exclude => [],
-        :start_new_files_at => :end
+        :start_new_files_at => :end,
+        :delimiter => "\n"
```

Comment:
Not sure if makes sense, but can we use here any of the separator constants defined in the File class?